Doc/library/urllib.request.rst

```diff
@@ -734,7 +734,7 @@ The following attribute and methods should only be used by classes derived from
 
    This method, if implemented, will be called by the parent
    :class:`OpenerDirector`.  It should return a file-like object as described in
-   the return value of the :meth:`open` of :class:`OpenerDirector`, or ``None``.
+   the return value of the :meth:`~OpenerDirector.open` method of :class:`OpenerDirector`, or ``None``.
```

Comment:
It is not related to this PR, but if some item is mentioned multiple times in one paragrafph, I prefer to keep link only in one of them (usually at the first occurrence). ``` :class:`!OpenerDirector` ``` or ``` ``OpenerDirector`` ```. Too many duplicated links do not make the text more readable.

Reply:
Makes sense, will keep it in mind for the future.